src/essreflectometry/amor/__init__.py

```diff
@@ -29,6 +29,7 @@
         conversions.providers,
         resolution.providers,
         beamline.providers,
+        data.providers,
```

Comment:
Can you remove this? If we insert this provider by default, then every time the user want to provide the file name in a different way, they first have to remove the pooch provider.

Reply:
Parameters override providers in Sciline so that is not a problem. See example below:
```python
import sciline as sl

def test(f: float) -> int:
    return int(f)

sl.Pipeline([test], params={int: 1})
```
Similarly when replacing providers:
```python
pl = sl.Pipeline([test], params={float: 1.0})
pl[int] = 2
pl.compute(int)
```
The above does not raise `AmbigiousProviderError`.
Or did I misunderstand you?

Reply:
I agree with @jl-wynen, this should be removed. The providers we use for the docs are not useful for normal users. If they forget to insert data (or a data provider) we want an *error*, not something that silently loads some unknown data and processes this. This could result in someone wasting hours debugging.

Reply:
It makes sense to exclude providers we only use in docs from the default provider list. I'll remove it.